_posts/en/2020-12/2020-12-16-welcome-to-the-new-rism-website.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "Welcome to the new RISM Website!"
+date: 2020-12-16
+lang: en
+post: true
+category: new_at_rism
+image: "/images/news/2020-12/neue_website_2020.png"
+---
+We invite you to take a look at the website of the [RISM Editorial Center](/editorial-center.html){:blank}, which has been redesigned in cooperation with the [RISM Digital Center](/digital-center.html){:blank}. 
```

Comment:
I am not 100% sure but I think it is better (mandatory?) to leave one empty line after the front-matter

Reply:
OK, I will add that to the wiki too.